libultraship/libultraship/Utils.cpp

```diff
@@ -1,9 +1,14 @@
 #include "Utils.h"
+#include <cstring>
+
+#ifdef _MSC_VER
+#define strdup _strdup
+#endif
 
 namespace Utils {
 	std::vector<std::string> SplitText(const std::string text, char separator = ' ', bool keep_quotes = false) {
 		std::vector<std::string> args;
-		char* input = _strdup(text.c_str());
+		char* input = strdup(text.c_str());
```

Comment:
Seems like this might be an area of win32/linux incompatibility

https://stackoverflow.com/questions/7582394/strdup-or-strdup

Reply:
I added a macro at the top of the file to redirect `strdup` to `_strdup` when using MSVC.